### ISSUE TITLE
Missing query string for url in bundle task

### ISSUE THREAD

**Issue by dotted** [mjohnston/react-native-webpack-server#73]
react-native 0.11.0 requires new query string to the url containing the bundle.

Currently getting this error when running bundle:

```
   Error: Error validating module options: child "platform" fails because ["platform" is required]
       at /.../react-native/packager/react-packager/src/lib/declareOpts.js:60:13
       at /.../react-native/packager/react-packager/src/Server/index.js:161:20
       at tryCallOne (/.../react-native/node_modules/promise/lib/core.js:37:12)
       at /.../react-native/node_modules/promise/lib/core.js:103:15
       at flush (/.../react-native/node_modules/promise/node_modules/asap/raw.js:50:29)
       at doNTCallback0 (node.js:408:9)
       at process._tickCallback (node.js:337:13)
   Error creating bundle... undefined
```

However changing

``` javascript
var url = 'http://localhost:' + opts.port + '/index.ios.bundle';
```

to

``` javascript
var url = 'http://localhost:' + opts.port + '/index.ios.bundle?platform=ios&dev=false'
```

makes the bundle succeed.

**Comment by naoisegolden** [comment]
Changing `./ios/MyApp/AppDelegate.m:34` to

```
jsCodeLocation = [NSURL URLWithString:@"http://localhost:8081/index.ios.bundle?platform=ios&dev=false"];
```

worked for me

**Comment by elliottsj** [comment]
Closed via #100, #102
